.gitignore

```diff
@@ -20,6 +20,7 @@ node_modules
 .env.development.local
 .env.test.local
 .env.production.local
+.soroban
```

Comment:
Can you refresh my memory: what was this is needed for?

Reply:
Hmm maybe it's not needed, not even sure how I actually generated a `.soroban` dir so let me get rid of that.

Reply:
removed in d172235d8ea857ddb35022b80367ea7dbba8ff71